packages/block-library/src/page-list/convert-to-links-modal.js

```diff
@@ -15,10 +16,18 @@ export function ConvertToLinksModal( { onClick, onClose, disabled } ) {
 			title={ __( 'Edit Page List' ) }
 			className="wp-block-page-list-modal"
 			aria={ {
-				describedby: 'wp-block-page-list-modal__description',
+				describedby: useInstanceId(
+					ConvertToLinksModal,
+					'wp-block-page-list-modal__description'
+				),
```

Comment:
We're not breaking these rules. Using hooks inline is perfectly fine, as long as the traditional rules of hook are not broken. That rule refers to passing the hook itself (as in the hook function value) around.

Reply:
I personally have strong feelings against unnecessary dis-location (the term I've coined for the opposite of co-location 😝) without good enough reasons. I'm open to being wrong, but it would be surprising if the linter complains about it, and if it does it seems like a clear bug or temporary limitation of the React compiler as there's no significant semantic difference between the two styles. But even if it's a bug, as long as the linter complains I'm 100% willing to go that route.

Can we maybe make sure of this before taking action? Do you have a reproduction I can look at at hand?

Reply:
Based on @tyxla's observations I think this is fine to merge as is. Can always go back and change it, if we realize this test was incorrect for some reason. Thanks for chiming in!

Reply:
Thanks for double-checking, @tyxla, and sorry for the false alarm, @DaniGuardiola!